### DISCUSSION TITLE
java.lang.NoClassDefFoundError: Could not initialize class io.atlasmap.v2.Json

### DISCUSSION THREAD

**Discussion by SujalaTangudu** [atlasmap/atlasmap#4055] [Q&A]
Dear Community,

Getting "java.lang.NoClassDefFoundError: Could not initialize class io.atlasmap.v2.Json" exception when trying to convert the the input instance to desired ouput instance with the relevant mapping spec. Please do provide any suggestions .

Thanks
Sujala

**Comment by SujalaTangudu** [comment]
Dear Community,

I want to add the additional information regarding the above exceptionThe above issue is being faced when I am trying to use data-mapper2.1.6 and deployed in aws environment.

Thanks 
Sujala

**Comment by igarashitm** [comment]
Can you explain more? what did you do, what do you want to do, full stacktrace, etc. Also I would suggest to use the latest version 2.3.18 and try locally before deploying in aws.
